src/components/movie-form/movie-form.tsx

```diff
@@ -146,11 +149,11 @@ export default function MovieForm({ onSubmit, params }: FormProps) {
         overview
         <textarea
           placeholder="Movie description"
-          {...(register(Overview), { required: false })}
+          {...register(FormField.Overview, { required: false })}
         />
       </label>
       <div>
-        <Button className="cancel modal-edit-button" type="reset">
+        <Button className="cancel modal-edit-button" onClick={() => reset()}>
```

Comment:
Let's move it to handler or you can pass just onClick={reset}

Reply:
Moved to handleReset handler.